src/app/Plans/components/Wizard/GeneralForm.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { Form, FormGroup, TextArea, TextInput, Title } from '@patternfly/react-core';
+import SimpleSelect, { OptionWithValue } from '@app/common/components/SimpleSelect';
+import { Provider } from '@app/Providers/types';
+
+interface IGeneralFormComponent {
+  sourceProviders: Provider[];
+  targetProviders: Provider[];
+}
+
+const GeneralForm: React.FunctionComponent<IGeneralFormComponent> = ({
+  sourceProviders,
+  targetProviders,
+}: IGeneralFormComponent) => {
+  const [planName, setPlanName] = React.useState<string>('');
+  const [planDesc, setPlanDesc] = React.useState<string>('');
+  const [srcProvider, setSrcProvider] = React.useState<Provider | null>(null);
+  const [tgtProvider, setTgtProvider] = React.useState<Provider | null>(null);
```

Comment:
Still a few more tgt abbreviations here.

Reply:
Agreed, let's use `sourceProvider` and `targetProvider` for readability